report_dynamic/README.rst

```diff
@@ -0,0 +1,32 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Report Dynamic
+=================
+
+Generate dynamic contracts based on Building Blocks/Section
+
+Installation
+============
+* Just Install
```

Comment:
Use the new `readme/*` files to autogenerate this, get them from the [OCA module template](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme). In the ROADMAP section, mention your point about the One2many field rendering.